test/algorithms/classifiers/test_qsvc.py

```diff
@@ -114,6 +116,39 @@ def test_with_kernel_parameter(self):
         with self.assertWarns(QiskitMachineLearningWarning):
             QSVC(kernel=1)
 
+    def test_unbound_user_params(self):
```

Comment:
The test name is slightly misleading. Is it a test that tests the whole process? If not, then there should be such a test.